src/docker/compose.local.yaml

```diff
@@ -1,51 +1,63 @@
 # yamllint disable rule:empty-lines rule:key-ordering rule:brackets
+# cspell:word localnet
 ---
 services:
   aptos-node:
     network_mode: 'host'
-    build:
-      context: '../../'
-      dockerfile: 'src/docker/aptos-node/Dockerfile'
+    image: 'econialabs/aptos-cli:4.1.0'
+    profiles: ["test-harness"]
     ports:
     - '50051:50051'
     - '8070:8070'
     - '8090:8090'
     - '8080:8080'
     - '8081:8081'
     healthcheck:
-      test: ['CMD', 'curl', '-f', 'http://localhost:8070/']
-      interval: '30s'
-      timeout: '10s'
-      retries: 5
-      start_period: '30s'
-    command: ['bash', '/app/run-fullnode.sh']
+      test: ['CMD', 'curl', 'http://localhost:8070/']
+      interval: '5s'
+      timeout: '5s'
+      retries: 10
+      start_period: '60s'
```

Comment:
This should go inside the docker image itself (using [`HEALTHCHECK`](https://docs.docker.com/reference/dockerfile/#healthcheck)). Imagine we use this in another repo, we'll have to copy pasta this there. Then imagine we want to change the `start_period` or `retries`, we'll have to make a PR into every repo where this is used and hope we didn't miss any.

Reply:
> This should go inside the docker image itself 

I would actually advise against health check internalization, because on AWS we configure the health checks externally like this.

Reply:
Also, I don't think we would deploy the aptos node

Reply:
My mistake, AWS considerations don't apply here (at least for now). Consider resolved on my end, I yield to @CRBl69 for final resolution

Reply:
Awesome, Bogdan and I discussed it on call on Friday, decided to keep it in to be explicit, but it's also in the base image as well. (As well as the updated healthcheck)